### PR TITLE
Clean up the lagom-java.g8 template

### PR DESCRIPTION
**Pull request by TimMoore**
- Fix deprecation warnings
- Remove empty header comments
- Organize imports
- Reformat to standard four-space indent
- Change a few hard-to-read line breaks and other minor formatting

### CODE REVIEW

**Comment by TimMoore** [comment]
It might be easier to review with whitespace ignored in the diff: https://github.com/lagom/lagom-java.g8/pull/28/files?w=1

**Comment by TimMoore** [comment]
The build is green now

File: src/main/g8/$name__norm$-api/src/main/java/$package$/api/$name__Camel$Service.java [hunk 1]
```diff
@@ -15,45 +8,48 @@
...
-
-  /**
-   * Example: curl -H "Content-Type: application/json" -X POST -d '{"message":
-   * "Hi"}' http://localhost:9000/api/hello/Alice
-   */
-  ServiceCall<GreetingMessage, Done> useGreeting(String id);
+    /**
+     * Example:
+     * curl -H "Content-Type: application/json" -X POST -d '{"message": "Hi"}' http://localhost:9000/api/hello/Alice
+     */
```
Comment (ignasi35) [inline src/main/g8/$name__norm$-api/src/main/java/$package$/api/$name__Camel$Service.java]: Thank you thank you thank you!!! :-)

File: src/main/g8/build.sbt [hunk 2]
```diff
@@ -45,6 +46,7 @@ lazy val `$name;format="norm"$-stream-impl` = (project in file("$name;format="no
     libraryDependencies ++= Seq(
       lagomJavadslPersistenceCassandra,
       lagomJavadslKafkaClient,
+      lagomLogback,
```
Comment (ignasi35) [inline src/main/g8/build.sbt]: Nice!

**Pull request merged**